example-next/pages/profile/[...members].js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable max-len */
+/*
+ TODO:
+   - [ ] determine dev/prod "paths" configuration for getStaticPaths (env var to set top z-index percentile on prod?)
+
+*/
+import axios from "axios";
+import {Title} from "@mantine/core";
+import {useTranslation} from "next-i18next";
+import {useRouter} from "next/router";
+import {serverSideTranslations} from "next-i18next/serverSideTranslations";
+import {Profile, NonIdealState} from "@datawheel/canon-next";
+import getProfile from "../../helpers/getProfile";
+// import {profileSearchConfig} from "../../helpers/search";
+const {NEXT_PUBLIC_CMS} = process.env;
+
+function ProfilePage({profile, formatters}) {
+  const router = useRouter();
+  const {t} = useTranslation("profile");
+
+  if (router.isFallback) return <NonIdealState />;
+  return (
+    <>
+      <Title>Profile</Title>
+      <Profile
+        formatters={formatters}
+        profile={profile}
+        linkify={profile => profile.reduce((href, member) => `${href}/${member.slug}/${member.memberSlug || member.id}`, "/profile")}
+        // you can also specify the configuration for ProfileSearch here:
+        // searchProps={{placeholder: "Seach profiles"}}
+        t={t}
+      />
+    </>
+  );
+}
+
+export default ProfilePage;
+
+
+export async function getStaticPaths() {
+  return {
+    paths: [
+      // Place here the paths you want to cache on build
+    ],
+    fallback: true
+  };
```

Comment:
Instead of requiring sites to implement the entirity of `getStaticPaths` (but I understand the need for this export to still exist in the file), can we export a helper function that accepts an optional paths argument? That way if we need to change some of these defaults, we won't necessarily require all sites to copy/paste the new settings:

```
return profileStaticPathsDefaults(); // I'm open to suggestions about a better/shorter function name
```